misc/extension_api_validation/4.4-stable.expected

```diff
@@ -329,3 +329,11 @@ Validate extension JSON: API was removed: classes/VisualShader/methods/set_graph
 Validate extension JSON: API was removed: classes/VisualShader/methods/get_graph_offset
 
 The graph_offset property was removed from the resource. This information is now stored in `vs_editor_cache.cfg`.
+
+
+GH-107952
+--------
+Validate extension JSON: API was removed: classes/ArrayMesh/methods/surface_get_format
+Validate extension JSON: API was removed: classes/ArrayMesh/methods/surface_get_primitive_type
+
+Moved methods from ArrayMesh to its parent Mesh class.  No adjustments should be necessary.
```

Comment:
This is definitely a breaking change, because function pointers are fetched by class name and hash:

https://github.com/godotengine/godot/blob/30456ba095c425d5ecde320a96a763afb93bbcb0/core/extension/gdextension_interface.h#L2867-L2879

This means the pointer can no longer be fetched under the old class name. Extensions compiled against earlier Godot APIs that don't explicitly check this would thus run into null pointer function calls (undefined behavior).

You can address this by registering compatibility methods that just pass through to the new implementation, and afterwards remove the changes to this file.